src/pages/TransactionIndexer/Tabs/BalanceChangeTab.tsx

```diff
@@ -0,0 +1,23 @@
+import EmptyTabContent from "../../../components/IndividualPageContent/EmptyTabContent";
+import {getCoinBalanceChanges} from "../utils";
+import {CoinBalanceChangeTable} from "./Components/CoinBalanceChangeTable";
+import {User_Transaction} from "../../../gql/graphql";
+
+type BalanceChangeTabProps = {
+  transaction: User_Transaction;
+};
+
+export default function BalanceChangeTab({transaction}: BalanceChangeTabProps) {
+  const balanceChanges = getCoinBalanceChanges(transaction);
+
+  if (balanceChanges.length === 0) {
+    return <EmptyTabContent />;
+  }
+
+  return (
+    <CoinBalanceChangeTable
+      balanceChanges={balanceChanges}
+      transaction={transaction}
+    />
+  );
+}
```

Comment:
Are all / most of these files just copy and pasted? This is going to make things hard to maintain. why not just modify the existing routes and components to use indexer?

Reply:
to create a fail-safe mechanism, if indexer source goes wrong & goes down, we need to fall back to FN again, so right now to keep things clean and separate, i have a separate folder for the txn from indexer, but, after couple weeks running & testing and things are determined to be good, i'll do the complete migration and remove the old one.